src/components/entity/list.vue

```diff
@@ -43,7 +43,7 @@ except according to the terms contained in the LICENSE file.
         type="entity"
         :top="pagination.size"
         :filter="odataFilter != null || !!searchTerm"
-        :total-count="dataset.dataExists ? dataset.entities : 0"/>
+        :total-count="dataset.dataExists && !pagination.page ? dataset.entities : 0"/>
```

Comment:
I don't want to show "Loading first N entities...", with this change it will always display "Loading entiities..." when there is a pagination information in the query parameter on the first loading. Note that on change pages using pagination control doesn't show any loading message.